lib/spark_api/models/search_template/quick_search.rb

```diff
@@ -5,7 +5,12 @@ class QuickSearch < Base
       include Concerns::Savable,
               Concerns::Destroyable
 
-      self.element_name="searchtemplates/quicksearches"
+      self.element_name = "searchtemplates/quicksearches"
+
+      def fields(args = {})
+        arguments = {:_expand => "Fields"}.merge(args)
```

Comment:
Yep, the initialize method doesn't assign ['Fields'] to @fields, so the check in the fields method is useless in this case.